force-app/main/default/classes/SOQL.cls

```diff
@@ -39,9 +42,10 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List <Contact> getContactsByFirstName(){
+        List<Contact> justJohns = [SELECT Id, Name FROM Contact WHERE FirstName = 'John'];
```

Comment:
For practice try moving the string John into a variable and using it in the SOQL